pkg/secrets/pushtofile/secret_spec.go

```diff
@@ -64,15 +66,21 @@ func (t *SecretSpec) unmarshalFromMap(node *yaml.Node) error {
 	}
 
 	// Mapping node but has multiple entries
-	if len(mapValue) != 1 {
-		return fmt.Errorf(invalidSecretSpecErr, node.Line)
-	}
 
+	t.ContentType = "text"
+	count := 0
 	for k, v := range mapValue {
-		t.Path = v
-		t.Alias = k
+		if k == "content-type" {
+			t.ContentType = v
+		} else {
+			count = count + 1
+			if count > 1 {
```

Comment:
I don't think any unit tests trigger this condition (too many mapping entries)

Reply:
The test TestNewSecretSpecs/secret_spec_map_with_multiple_keys triggers this condition.